### PR TITLE
Add normalize and fingerprint

### PR DESCRIPTION
**Pull request by ehfeng**
Effectively a duplicate of https://github.com/alculquicondor/psqlparse/pull/18. Given that it's been 8 months and there are conflicts, I went ahead and copied it, added test, and added the fingerprint method as well.

### CODE REVIEW

**Comment by alculquicondor** [comment]
Sorry, but I don't see any fingerprint method.

**Comment by ehfeng** [comment]
Accidentally pushed commit to wrong branch. Added now @alculquicondor

File: test/test_fingerprint.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class FingerprintTest(unittest.TestCase):
+    def test_simple(self):
+        query1 = "SELECT * FROM mytable WHERE col1 = 1"
+        query2 = "SELECT * FROM mytable WHERE col1 = 2"
+        query3 = """SELECT *
+                    FROM mytable
+                    WHERE col1 = 2"""
+        assert fingerprint(query1) == fingerprint(query2)
```
Comment (alculquicondor) [inline test/test_fingerprint.py]: use self.assertEqual

File: test/test_normalize.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+from psqlparse import normalize
+from psqlparse.exceptions import PSqlParseError
+from psqlparse import nodes
+
+
+class NormalizeTest(unittest.TestCase):
+    def test_simple(self):
+        query = "SELECT * FROM mytable WHERE col1 = 1"
+        assert normalize(query) == "SELECT * FROM mytable WHERE col1 = $1"
```
Comment (alculquicondor) [inline test/test_normalize.py]: use self.assertEqual

**Comment by vharitonsky** [comment]
Also added fingerprinting in #44

**Comment by ehfeng** [comment]
@alculquicondor updated

**Comment by alculquicondor** [comment]
PEP8 failed

**Comment by ehfeng** [comment]
Closing, as I probably won't be able to finish this.

**Pull request closed without merging**